src/extension-runners/firefox-android.js

```diff
@@ -122,15 +123,29 @@ export class FirefoxAndroidExtensionRunner {
       adbHost,
       adbPort,
       ADBUtils = DefaultADBUtils,
+      cleanArtifacts,
```

Comment:
It isn't clear to me who should be passing this cleanArtifacts option (it seems that it is not actually being used in the current version of this patch), was it meant to be a new command line option?

Reply:
Yes, This needs to be correctly called and used. Will use implementation of current cli options used.